<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_MHWD</class>
 <widget class="QWidget" name="Page_MHWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Driver</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Free</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Installed</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowAll">
       <property name="text">
        <string>Show all devices</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonInstallFree">
       <property name="text">
        <string>Auto Install
Free Driver</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonInstallNonFree">
       <property name="text">
        <string>Auto Install
Non-Free Driver</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
